jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -47,7 +47,7 @@
 import com.ibm.oti.vm.VM;
 import com.ibm.oti.vm.VMLangAccess;
 
-/*[IF Sidecar19-SE]
+/*[IF Sidecar19-SE]*/
```

Comment:
why this change?

Reply:
Doesn't that just move the syntax error from 9 to 8?

Reply:
No: this change eliminates most of the syntax errors in the unprocessed code.